samples/bluetooth/peripheral_proximity/Kconfig

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "Kconfig.zephyr"
+
+menu "Nordic proximity sample"
```

Comment:
I don't think we need this file, we don't seem to be declaring new sample-specific options